fbw-a380x/src/base/flybywire-aircraft-a380-842/SimObjects/AirPlanes/FlyByWire_A380_842/model/A380_COCKPIT.xml

```diff
@@ -5481,6 +5472,34 @@
             </UseTemplate>
         </Component>
 
+        <Component ID="EFB_CPT_MOUNT" Node="EFB_CPT_MOUNT">
+            <UseTemplate Name="ASOBO_GT_Visibility_Code">
+                <VISIBILITY_CODE>(L:A380X_CPT_EFB_HIDDEN, bool) 0 ==</VISIBILITY_CODE>
+            </UseTemplate>
+        </Component>
+
+        <Component ID="LH_SLIDING_LIGHTING_KNOB" Node="LH_SLIDING_LIGHTING_KNOB">
+            <UseTemplate Name="ASOBO_GT_MouseRect">
+                <CALLBACKCODE_DEFAULT_IM>(M:Event) 'LeftSingle' scmp 0 == if{ (L:A380X_CPT_EFB_HIDDEN, bool) ! (&gt;L:A380X_CPT_EFB_HIDDEN, bool) }</CALLBACKCODE_DEFAULT_IM>
+                <MOUSEFLAGS>LeftSingle</MOUSEFLAGS>
+                <CURSOR>Hand</CURSOR>
+            </UseTemplate>
```

Comment:
You don't normally use such a low level template directly as it doesn't setup all the interaction stuff like tooltips and actions.